packages/core/src/base/style/BaseStyle.js

```diff
@@ -131,17 +131,15 @@ const css = ({ dt }) => `
     clip: rect(0 0 0 0);
     height: 1px;
     margin: -1px;
+    opacity: 0;
     overflow: hidden;
     padding: 0;
+    pointer-events: none;
     position: absolute;
+    white-space: nowrap;
     width: 1px;
 }
 
-.p-hidden-accessible input,
-.p-hidden-accessible select {
-    transform: scale(0);
-}
-
```

Comment:
This must be removed as it hides the input from NVDA because it won't read anything of size 0 . I think this was a legacy PrimeFaces fix that is no longer needed